DependencyInjection/MonologExtension.php

```diff
@@ -468,6 +468,8 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 $handler['bubble'],
                 $handler['use_ssl'],
                 $handler['message_format'],
+                $handler['host'],
```

Comment:
@stof this sort of comment is really not helpful IMO, and I don't need to get fact-checking of hypothetical statements in my inbox :/